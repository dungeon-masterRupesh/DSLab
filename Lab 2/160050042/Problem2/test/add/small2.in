3 4
5 2 3 4 
0 0 5 3 
5 2 0 0 
3 4
3 5 2 2 
1 3 3 1 
1 4 4 5 
