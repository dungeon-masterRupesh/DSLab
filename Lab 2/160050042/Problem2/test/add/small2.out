3 4
8 7 5 6 
1 3 8 4 
6 6 4 5 
